.github/workflows/release.yaml

```diff
@@ -54,13 +58,33 @@ jobs:
           RELEASE: ${{ env.RELEASE_VERSION }}-ARM
         run: |
           cd opensearch-operator
-          pwd
-          ls
-          go version
-          go env
           aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws/opsterio
           docker build -f Dockerfile-ARM -t public.ecr.aws/opsterio/opensearch-operator:$RELEASE ./
           docker push public.ecr.aws/opsterio/opensearch-operator:$RELEASE
           docker tag public.ecr.aws/opsterio/opensearch-operator:$RELEASE public.ecr.aws/opsterio/opensearch-operator:latest-ARM
           docker push public.ecr.aws/opsterio/opensearch-operator:latest-ARM
-
+      - name: Copy-crd-to-chart-dir
+        run: |
+          cd opensearch-operator
+          make manifests
```

Comment:
`make manifests` was already run in an earlier step, no need to run it twice

Reply:
agree